zetaclient/bitcoin/bitcoin_client.go

```diff
@@ -43,15 +46,15 @@ type BTCLog struct {
 	WatchGasPrice zerolog.Logger
 }
 
-// BitcoinChainClient represents a chain configuration for Bitcoin
+// ChainClient represents a chain configuration for Bitcoin
 // Filled with above constants depending on chain
-type BitcoinChainClient struct {
-	*ChainMetrics
+type ChainClient struct {
```

Comment:
Do you want to keep the name `BitcoinChainClient` to differentiate a bit with the interface name `ChainClient`?

Reply:
it is already in another package, we would also get lint errors saying that the name is redundant.

Reply:
the `make lint` worked well on my end. strange